streamdeck_ui/api.py

```diff
@@ -28,17 +28,6 @@ class KeySignalEmitter(QObject):
 streamdesk_keys = KeySignalEmitter()
 
 
-def _replace_special_keys(key):
-    """Replaces special keywords the user can use with their character equivalent."""
-    if key.lower() == "plus":
-        return "+"
-    if key.lower() == "comma":
-        return ","
-    if key.lower() == "delay":
-        return "delay"
-    return key
-
-
```

Comment:
Seems to be unused leftovers from 28630d9c3eb0557627d954e50b60a0099a2ec602

Reply:
Thanks, you are correct, should have been removed.